### ISSUE TITLE
Cannot Reinstall or Uninstall

### ISSUE THREAD

**Issue by jonsq** [phpmanager/phpmanager#37]
My computer is running Windows 10 Pro with IIS 10.

I installed version 2.4 a few days ago.  For unrelated reasons, I had to reinstall IIS 10.  After reinstalling IIS, the PHPManager was no longer showing in the IIS manager.  I tried to repair the install and it failed.  I tried to uninstall and it also failed.  Currently I'm unable to repair or uninstall PHPManager 2.4.  

Here is a verbose output log file that was generated via msiexec during my most recent uninstall attempt:
[output.log](https://github.com/phpmanager/phpmanager/files/4150500/output.log)

Here's the PHPMangager Installer's self generated log file:
[Web.Management.PHP.PowerShell.InstallLog.log](https://github.com/phpmanager/phpmanager/files/4150511/Web.Management.PHP.PowerShell.InstallLog.log)

Can anyone offer any advise?

**Comment by lextm** [comment]
The actual error is

```
MSI (s) (94:C0) [16:34:59:802]: Executing op: ActionStart(Name=UnInstallPHPSetup,,)
Action 16:34:59: UnInstallPHPSetup. 
MSI (s) (94:C0) [16:34:59:804]: Executing op: CustomActionSchedule(Action=UnInstallPHPSetup,ActionType=3073,Source=BinaryData,Target=ManagedInstall,CustomActionData=/installtype=notransaction /action=uninstall /LogFile= "C:\Program Files\PHP Manager 2 for IIS\Web.Management.PHP.Setup.dll" "C:\Program Files\PHP Manager 2 for IIS\ConfigFile.xml")
MSI (s) (94:34) [16:34:59:807]: Invoking remote custom action. DLL: C:\WINDOWS\Installer\MSID793.tmp, Entrypoint: ManagedInstall
Error 1001. Error 1001. Exception occurred while initializing the installation:
System.Configuration.ConfigurationErrorsException: Configuration system failed to initialize.
```
which links to https://github.com/phpmanager/phpmanager/blob/5390f2eea6e105cc2e9c2571eecd4b08fa05092f/Setup/Product.wxs#L260

You might try to manually unregister that dll using installutil (reverse of #33).

**Comment by jonsq** [comment]
Sorry for the delayed response.

I tried your suggestion, (unregister that dll), on Feb 4, and it did not help at that time.

However, I just re-attempted running the 2.4 installer today and it worked w/o issue.  

I don't know what changed on my PC, but I do know that I installed Visual Studio 2019 sometime between Feb 4, and today.  That's the only thing that comes to mind that could have possibly changed a system setting that fixed the PhpManager installer issue.

Wish I could report exactly what fixed the issue, but honestly I'm just happy it's working now.
